backend/nginx.conf

```
@@ -0,0 +1,15 @@
+upstream django {
+ server djangoindia:8000;
```

If you see the docker-compose file, our backend container is running at port 8000 and its name is `djangoindia-backend` and since our nginx container will send request to our backend we have to pass the backend container name here.
```suggestion
 server djangoindia-backend:8000;
```